package.json

```diff
@@ -343,6 +343,7 @@
         "express": "^4.16.4",
         "fs-extra": "^9.0.0",
         "md5-file": "^4.0.0",
-        "rimraf": "^3.0.2"
+        "rimraf": "^3.0.2",
+        "sanitize-filename": "^1.6.3"
```

Comment:
Did you end up using this dependency?

Reply:
I see, it is potentially useful to solve other issues, remove it in this PR, and introduce it later to solve the other issues if it really helps.